MonoGame.Framework/Input/Mouse.cs

```diff
@@ -139,20 +94,21 @@ public static MouseState GetState(GameWindow window)
 
 #elif DESKTOPGL || ANGLE
 
-            var state = OpenTK.Input.Mouse.GetCursorState();
-            var pc = ((OpenTKGameWindow)window).Window.PointToClient(new System.Drawing.Point(state.X, state.Y));
-            window.MouseState.X = pc.X;
-            window.MouseState.Y = pc.Y;
+            int x, y;
+
+            var state = SDL.SDL_GetGlobalMouseState(out x, out y);
+            var pos = PrimaryWindow.Position;
+
+            window.MouseState.X = x - pos.X - BorderX;
+            window.MouseState.Y = y - pos.Y - BorderY;
```

Comment:
@tomspilman This is another position where I needed border size, the reason why it's needed here is so we can track global mouse.

We can track mouse while it's inside window with `SDL.SDL_GetMouseState`, but as soon as it gets out it won't be tracked.